modules/auxiliary/gather/ldap_query.rb

```diff
@@ -206,6 +206,21 @@ def output_data_csv(entries)
     generate_rex_tables(entries, 'csv')
   end
 
+  def normalize_entries(entries)
```

Comment:
Not a blocker; Does it make sense to make the GUID field human readable?

Reply:
See comment below. This is planned for a future PR which will be worked on ASAP after this is landed.